examples/acados_matlab_octave/pendulum_on_cart_model/example_ocp.m

```diff
@@ -212,7 +215,7 @@
 ocp_opts.set('qp_solver_cond_ric_alg', qp_solver_cond_ric_alg);
 ocp_opts.set('qp_solver_warm_start', qp_solver_warm_start);
 ocp_opts.set('qp_solver_iter_max', qp_solver_max_iter);
-if (~isempty(strfind(qp_solver, 'partial_condensing')))
+if (contains(qp_solver, 'partial_condensing'))
```

Comment:
I think this is not supported in Octave, thats why I would undo this change.

Reply:
I understand, sorry for the mistake.